### ISSUE TITLE
BUG: Unhandled error in plugin discover: No JSON object could be decoded

### ISSUE THREAD

**Issue by chemondelay** [Flexget/Flexget#1698]
### Steps to reproduce:
Use `discover` with BTN search plugin.

#### Config:
```
  download-series:
    template: deluge
    discover:
      what:
        - next_series_episodes:
            from_start: yes
      from:
        - btn: '{? btn.api ?}'
    series:
      all:
        
        - "Adventure Time":
            set:
              series_title: "Adventure Time"
              series_year: 2010
            from_group: NTb
            quality: 1080p webdl

        - "Crazy Ex-Girlfriend":
            set:
              series_title: "Crazy Ex-Girlfriend"
              series_year: 2015
            from_group: TrollHD
            quality: 1080p

        - "It's Always Sunny in Philadelphia":
            set:
              series_title: "It's Always Sunny in Philadelphia"
              series_year: 2005
            quality: 720p webdl

        - "Legion":
            set:
              series_title: "Legion"
              series_year: 2017
            from_group: KiNGS
            quality: 1080p webrip
        
        - "Man Seeking Woman":
            set:
              series_title: "Man Seeking Woman"
              series_year: 2015
            quality: 720p webdl
            
        - "Portlandia":
            set:
              series_title: "Portlandia"
              series_year: 2011
            quality: 720p webdl

        - "The Walking Dead":
            set:
              series_title: "Walking Dead, The"
              series_year: 2010
            from_group: CasStudio
            quality: 1080p webrip

        # - "Atlanta":
        #     set:
        #       series_name: "Atlanta"
        #       series_year: 2016

        # - "Better Call Saul":
        #     set:
        #       series_name: "Better Call Saul"
        #       series_year: 2015
        
        # - "The Magicians (2015)":
        #     set:
        #       series_name: "Magicians, The"
        #       series_year: 2015

        # - "Santa Clarita Diet":
        #     set:
        #       series_name: "Santa Clarita Diet"
        #       series_year: 2017

      settings:
        all:
          identified_by: ep
    deluge:
      label: series
      addpaused: no
      queuetotop: yes
      movedone: /mnt/media/series/{{ series_title }} ({{ series_year }})/Season {{ series_season }}
```
  
#### Log:
```
2017-02-18 14:31 CRITICAL task          download-series BUG: Unhandled error in plugin discover: No JSON object could be decoded
Traceback (most recent call last):
 File "/usr/lib/python2.7/site-packages/flexget/task.py", line 483, in __run_plugin
   return method(*args, **kwargs)
 File "/usr/lib/python2.7/site-packages/flexget/event.py", line 23, in __call__
   return self.func(*args, **kwargs)
 File "/usr/lib/python2.7/site-packages/flexget/plugins/input/discover.py", line 290, in on_task_input
   return self.execute_searches(config, entries, task)
 File "/usr/lib/python2.7/site-packages/flexget/plugins/input/discover.py", line 155, in execute_searches
   search_results = search.search(task=task, entry=entry, config=plugin_config)
 File "/usr/lib/python2.7/site-packages/flexget/plugins/sites/btn.py", line 59, in search
   content = r.json()
 File "/usr/lib/python2.7/site-packages/requests/models.py", line 866, in json
   return complexjson.loads(self.text, **kwargs)
 File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
   return _default_decoder.decode(s)
 File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
   obj, end = self.raw_decode(s, idx=_w(s, 0).end())
 File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
   raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
```

### Additional information:

- Flexget Version: `2.10.0`
- Python Version: `2.7.13`
- Installation method: `pip`
- OS and version: `linux 4.9.6-1`
- Link to crash log: http://pastebin.com/CbzBdyBW

**Comment by binamov** [comment]
I'm seeing the same

**Comment by cvium** [comment]
FYI, while flexget should not crash, this is sort of out of our hands. BTN api is down.

**Comment by crawln45** [comment]
btnapp.net domain expired. BTN is in the process of migrating the API to a new domain.

**Comment by binamov** [comment]
the fix seems to be to replace `api.btnapps.net` with `api.broadcasthe.net` in https://github.com/Flexget/Flexget/blob/develop/flexget/plugins/sites/btn.py @crawln45

**Comment by cvium** [comment]
The domain has been updated now following an official update from BTN staff. It will be available in version 2.10.5.
